cvat-canvas/src/typescript/canvasView.ts

```diff
@@ -1481,6 +1488,9 @@ export class CanvasViewImpl implements CanvasView, Listener {
 
     private addRect(points: number[], state: any): SVG.Rect {
         const [xtl, ytl, xbr, ybr] = points;
+        if (typeof (state.annotation_type) === 'undefined') {
```

Comment:
How is it possible?

Reply:
The same question in two cases below

Reply:
In some cases, especially in case of tracks, the value becomes `UNDEFINED` for subsequent frames.